src/Report/TextReporter.php

```diff
@@ -84,22 +82,28 @@ private function getDiff($isDecoratedOutput, array $fixResult)
             return '';
         }
 
-        $template = '';
-
         if ($isDecoratedOutput) {
-            $template .= '<comment>      ---------- begin diff ----------</comment>';
-            $template .= PHP_EOL.'%s'.PHP_EOL;
-            $template .= '<comment>      ----------- end diff -----------</comment>';
+            $template = '<comment>      ---------- begin diff ----------</comment>%s<comment>      ----------- end diff -----------</comment>';
+            $diff = implode(
+                PHP_EOL,
+                array_map(
+                    function ($string) {
+                        $string = preg_replace('/^(\+){3}/', '<info>+++</info>', $string);
```

Comment:
Hmmm indeed, we should be OK for now but I see your point.
Are these string `diff` formats (like the one used on default by `git`) formalized in a way?